properties/size/breakpoint.json

```diff
@@ -0,0 +1,26 @@
+{
+  "size": {
+    "breakpoint": {
+      "sm": {
+        "value": "576",
+        "comment": "small devices, landscape phones",
```

Comment:
Can we indicate in the comments that these are meant to be the min-widths? So, something like... "small devices, landscape phones: 576px and up"

Reply:
Sure. This will also be documented in storybook.

Reply:
Great! Thanks!